From 872409857351f28b1e3c21526bfa6606c918b176 Mon Sep 17 00:00:00 2001
From: Nikos Mavrogiannopoulos <nmav@gnutls.org>
Date: Sat, 19 Apr 2014 15:04:54 +0200
Subject: [PATCH] do not override gnutls' allocation functions

That was not being done using the API, and overriding them
is no longer possible in 3.3.x.
---
 guile/src/core.c |    5 -----
 1 files changed, 0 insertions(+), 5 deletions(-)

diff --git a/guile/src/core.c b/guile/src/core.c
index 6940eae..b40e93e 100644
--- a/guile/src/core.c
+++ b/guile/src/core.c
@@ -3364,11 +3364,6 @@ scm_init_gnutls (void)
 #include "core.x"
 
   /* Use Guile's allocation routines, which will run the GC if need be.  */
-  gnutls_malloc = scm_malloc;
-  gnutls_realloc = scm_realloc;
-  gnutls_secure_malloc = scm_malloc;
-  gnutls_free = free;
-
   (void) gnutls_global_init ();
 
   scm_gnutls_define_enums ();
-- 
1.7.1

